lib/ledger_sync/adaptors/quickbooks_online/bill/operations/create.rb

```diff
@@ -0,0 +1,65 @@
+module LedgerSync
+  module Adaptors
+    module QuickBooksOnline
+      module Bill
+        module Operations
+          class Create < Operation::Create
+            class Contract < LedgerSync::Adaptors::Contract
+              params do
+                required(:ledger_id).value(:nil)
+                required(:vendor).hash(Types::Reference)
+                required(:account).hash(Types::Reference)
```

Comment:
I think this field is optional no? As are due date, transaction date, and due memo

Reply:
Added dates and memo as optional, but relationship is _optional_ by default. Our schema doesn't validate required relationship atm.

Reply:
Ah ok...well should we just make it optional anyway so that when (If?) it is required, this won't fail?